src/devices/Mfrc522/Mfrc522.cs

```diff
@@ -34,6 +34,15 @@ public class MfRc522 : CardTransceiver, IDisposable
         /// </summary>
         public const SpiMode DefaultSpiMode = SpiMode.Mode0;
 
+        /// <inheritdoc/>
+        public override uint MaximumReadSize => 62;     // 64-byte FIFO, 2 byte CRC
```

Comment:
maybe make those 3 new properties static?
They can then be used easily to understand if a protocol is supported. They are always the same regardless of the instance

Reply:
An interface can have static virtual properties, but I don't think an abstract class can. They could be defined as static properties in each specific Transceiver class, but the compiler won't be able to enforce consistency - possibly leading to unintentional errors such as misspellings.

One reason I made them instance properties is that the card types are constructed with a reference to the CardTransceiver. Then, for instance, UltralightCommand can check whether the response to a ReadFast is too big by comparing it against _rfid.MaximumReadSize

OK to keep them as-is?

Reply:
Actually, I think SupportedProtocols makes more sense as a const (or static property), so I made that change. However, I think it is preferable to keep the maximum read/write size as a virtual property. OK?

Reply:
I thought that ```static abstract``` was only for interfaces - and ```CardTransceiver``` is an abstract class, not an interface. As I recall, I tried it anyway, and I was rewarded with errors from the compiler. I Did I miss something?